app/views/coronavirus_local_restrictions/show.html.erb

```diff
@@ -1,22 +1,33 @@
-<div class="govuk-width-container">
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-      <% content_for :breadcrumbs do %>
-        <%= render 'govuk_publishing_components/components/breadcrumbs', {
-          breadcrumbs: breadcrumbs,
-          collapse_on_mobile: true
-        } %>
-      <% end %>
+<% content_for :breadcrumbs do %>
+  <%= render 'govuk_publishing_components/components/breadcrumbs', {
+    breadcrumbs: breadcrumbs,
+    collapse_on_mobile: true
+  } %>
+<% end %>
 
-      <% content_for :title, t("coronavirus_local_restrictions.lookup.title") %>
+<% content_for :title, t("coronavirus_local_restrictions.lookup.title") %>
 
-      <%= render "govuk_publishing_components/components/title", title: t("coronavirus_local_restrictions.lookup.title") %>
+<% content_for :meta_tags do %>
+  <meta property="og:type" content="website" />
+  <meta property="og:site_name" content="GOV.UK">
+  <meta property="og:url" content="<%= request.base_url + request.path %>">
+  <meta property="og:title" content="<%= t("coronavirus_local_restrictions.lookup.meta_title") %>">
+  <meta property="og:description" content="<%= t("coronavirus_local_restrictions.lookup.meta_description") %>">
+<% end %>
 
-      <%= render "govuk_publishing_components/components/inset_text", text: t('coronavirus_local_restrictions.lookup.inset_text') %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/title", title: t("coronavirus_local_restrictions.lookup.title") %>
 
-      <%= sanitize(t("coronavirus_local_restrictions.lookup.body_content")) %>
+    <%= render "govuk_publishing_components/components/inset_text", text: t('coronavirus_local_restrictions.lookup.inset_text') %>
 
-      <%= render "shared/postcode-lookup" %>
-    </div>
+    <%= sanitize(t("coronavirus_local_restrictions.lookup.body_content")) %>
+    <%= form_for :postcode_lookup, :url => find_coronavirus_local_restrictions_path do |f| %>
+      <%= render "shared/postcode-lookup", {
+        error_message: error_message,
```

Comment:
Is this level of error message necessary? There's only one field in this form?

<img width="637" alt="Screenshot 2020-10-12 at 17 10 29" src="https://user-images.githubusercontent.com/861310/95768234-e96c5a80-0cad-11eb-9611-136042219a85.png">

Reply:
Page-level one is supposed to be at the top of the page, not directly above the field. We're aware it's in the wrong place and is on our list of things to fix after go live (this evening! 😱 )